frontend/src/containers/MetricsStep/Advance/Advance.tsx

```diff
@@ -10,6 +10,7 @@ import { useAppSelector } from '@src/hooks';
 import { useState } from 'react';
 
 export const Advance = () => {
+  const url = 'https://github.com/au-heartbeat/Heartbeat/blob/main/README.md';
```

Comment:
@TingyuDong 
should use this link: https://github.com/thoughtworks/HeartBeat/blob/main/README.md